.github/BOTMETA.yml

```diff
@@ -648,7 +648,7 @@ files:
   $modules/ip_netns.py:
     maintainers: bregman-arie
   $modules/ipa_:
-    maintainers: $team_ipa
+    maintainers: $team_ipa jwbernin
```

Comment:
Thanks for stepping up as a maintainer of the IPA modules! Please add yourself to `team_ipa` instead of here directly.

If you just want to be a maintainer of the ipa_dnsrecord module, add a separate entry specifically for that module with `maintainers: $team_ipa jwbernin`.